charts/generic-govuk-app/templates/nginx-configmap.yaml

```diff
@@ -168,6 +168,11 @@ data:
         location = /readyz {
           return 200 'ok\n';
         }
+
+
+        {{- if .Values.nginxConfigMap.extraConf }}
```

Comment:
Apologies if you've already considered this and it's doesn't fly for whatever reason, but wouldn't it be simpler/cleaner to just pass the actual extra config here instead of going to all the trouble of creating and including a separate file?

Reply:
Ahah yeah 😅 I think I was probably over thinking it bit. I was worried if the pattern got used other apps, which had much more config. I'll do another PR just passing it in via a Value.

Reply:
Ah ok cool, no probs!